.github/workflows/cla-assistant.yaml

```diff
@@ -13,16 +13,14 @@ jobs:
     steps:
       - name: "CLA Assistant"
         if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the Contributor License Agreement and I hereby accept the Terms.') || github.event_name == 'pull_request_target'
-        uses: cla-assistant/github-action@v2.1.3-beta
+        uses: cla-assistant/github-action@v2.3.0
         env:
-          GITHUB_TOKEN: ${{ github.token }}
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           PERSONAL_ACCESS_TOKEN : ${{ secrets.REPO_BOT_ACCESS_TOKEN }}
         with:
-          remote-organization-name: oqc-community
-          remote-repository-name: qat
           branch: 'bot/data'
           path-to-signatures: 'cla.json'
           custom-allsigned-prcomment: 'All Contributors have signed the CLA.'
           custom-pr-sign-comment: 'I have read the Contributor License Agreement and I hereby accept the Terms.'
-          allowlist: user1,bot*
-          path-to-document: 'https://github.com/oqc-community/qat/blob/main/contributor_license_agreement.md'
\ No newline at end of file
+          allowlist: chemix-lunacy,keriksson-rosenqvist,jfriel-oqc,owen-oqc,hamidelmaazouz,harry-oqc,bot*
```

Comment:
We could also just agree when we do our first PR rather than hard-coding it, either/or.

Reply:
You may need to do it for every PR. I'm not entirely sure. :thinking:

Reply:
Wat, I dearly hope it dosen't because if it does we'll have to find another bot to do so.

Reply:
Oh, I'll port this to the dev branch when it';s all verified as owrking too, so don't worry about that.